bpython/test/test_autocomplete.py

```diff
@@ -245,12 +245,12 @@ def test_att_matches_found_on_old_style_instance(self):
                                              locals_={'a': OldStyleFoo()}),
                             set(['a.method', 'a.a', 'a.b']))
         self.assertIn(u'a.__dict__',
-                      self.com.matches(3, 'a._', locals_={'a': OldStyleFoo()}))
+                      self.com.matches(3, 'a.__', locals_={'a': OldStyleFoo()}))
```

Comment:
Oh, my bad. The it totally makes sense to change it to 4. I have done that in 9f3460b.